causalml/inference/meta/explainer.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import numpy as np
+from eli5.sklearn import PermutationImportance
+import shap
+import matplotlib.pyplot as plt
+from lightgbm import LGBMRegressor
+from copy import deepcopy
+
+VALID_METHODS = ('gini', 'permutation', 'shapley')
+
+
+class Explainer(object):
+    def __init__(self, method, control_name, X, tau, classes, model_tau=None,
+                 features=None, normalize=True, override_checks=False, r_learners=None):
```

Comment:
@yungmsh,  Could you provide docstring for the class and member functions?

Reply:
Ah yes will do - docstrings are included in the individual learner scripts but I'll add them to the Explainer class as well.